lib/resolvers/relay.js

```diff
@@ -23,7 +23,7 @@ export function resolveRelayConnection(obj, args, context, info, type) {
   const { type: nodeType } = unwrapType(edgeType.getFields().node.type);
   const records =
     obj && obj[info.fieldName] && obj[info.fieldName].models
-      ? adaptRecords(obj[info.fieldName].models, nodeType.name)
+      ? adaptRecords(obj[info.fieldName].models)
```

Comment:
Note to self: Double check that these records don't need to be filtered by the GraphQL args.